articles/reactive-toarray.md

```diff
@@ -4,3 +4,5 @@ title: ToArray
 ---
 
 ![Marble diagram](~/images/reactive-toarray.svg)
+
```

Comment:
might be a good idea to mention that the sequence will just keep on accumulating in memory until it finishes.